extensions/amp-analytics/0.1/variables.js

```diff
@@ -161,10 +166,12 @@ export class VariableService {
     this.register_('$IF',
         (value, thenValue, elseValue) => value ? thenValue : elseValue);
     this.register_('$REPLACE', replaceMacro);
+    this.register_('LINKER_PARAM', (name, id) =>
```

Comment:
I like that we register `LINKER_PARAM` to variableService. 
Just a heads up that we are going to refactor the variableService to make it a doc level service. I think LinkerReader should be a window level service, we need to be extra careful because FIE and parent doc share the same doc, but not the same window. Shadow doc and parent doc share the same window but not the same doc...

Reply:
Thanks for the heads up. I added a TODO. Let me know if you think anything else is necessary here.